test/gridfs/README.md

```diff
@@ -0,0 +1,18 @@
+## GridFS Test Notes
```

Comment:
would make sense to run these tests on the GitHub actions CI as well (adjust ci.sh)